diesel/src/query_dsl/save_changes_dsl.rs

```diff
@@ -32,15 +31,17 @@ impl<'a, T, ST, Conn> SaveChangesDsl<Conn, ST> for &'a T where
 
 #[cfg(feature = "sqlite")]
 use sqlite::{SqliteConnection, Sqlite};
+#[cfg(feature = "sqlite")]
+use associations::Identifiable;
 
 #[cfg(feature = "sqlite")]
-impl<'a, T, ST> SaveChangesDsl<SqliteConnection, ST> for &'a T where
+impl<T, ST> SaveChangesDsl<SqliteConnection, ST> for T where
     Sqlite: HasSqlType<ST>,
-    T: Identifiable,
-    T::Table: FindDsl<&'a T::Id, SqlType=ST>,
-    &'a T: AsChangeset<Target=T::Table> + IntoUpdateTarget<Table=T::Table>,
-    Update<&'a T, &'a T>: ExecuteDsl<SqliteConnection>,
-    Find<T::Table, &'a T::Id>: LoadDsl<SqliteConnection, SqlType=ST>,
+    T: Copy + Identifiable,
+    T: AsChangeset<Target=<T as HasTable>::Table> + IntoUpdateTarget,
+    T::Table: FindDsl<T::Id, SqlType=ST>,
+    Update<T, T>: ExecuteDsl<SqliteConnection>,
+    Find<T::Table, T::Id>: LoadDsl<SqliteConnection, SqlType=ST>,
```

Comment:
That looks confusing, but I guess if it compiles it's fine (rustc is better at this than I am) 😄 What does it mean for T to be required to be Copy here? (Though I guess this was implicitly required before as well.)